_docs-sources/pipelines/using-pipelines/index.md

```diff
@@ -44,13 +44,27 @@ The following is an example of the recommended settings for branch protection:
    - On Failure, a new GitHub issue is created describing the failure. A new PR must be created to resolve any failures.
 
-## GitHub Enterprise Users
+## Allowing Pipelines Actions in GitHub Actions
 
-Gruntwork Pipelines uses a set of Gruntwork built re-usable Github Actions. Companies using GitHub Enterprise may need to explicitly allow Actions from Gruntwork to run in their GitHub organization. See the "Allow specified actions and reusable workflows" section in [Allowing select actions and reusable workflows to run](https://docs.github.com/en/enterprise-cloud@latest/admin/policies/enforcing-policies-for-your-enterprise/enforcing-policies-for-github-actions-in-your-enterprise#allowing-select-actions-and-reusable-workflows-to-run) to learn more.
+Gruntwork Pipelines uses a set of Gruntwork built re-usable Github Actions, which are available in the GitHub Actions marketplace. Gruntwork is a Verified Creator of GitHub Actions.
 
-You will need to allow the following Actions to run:
-- [pipelines-dispatch](https://github.com/gruntwork-io/pipelines-dispatch)
-- [pipelines-orchestrate](https://github.com/gruntwork-io/pipelines-orchestrate)
-- [pipelines-execute](https://github.com/gruntwork-io/pipelines-execute)
+### GitHub Enterprise
 
-Navigate to each repository to retrieve the latest tagged release for each action.
+Gruntwork recommends explicitly allowing Actions required by Gruntwork Pipelines to run in your GitHub organization. See [Allowing select actions and reusable workflows to run](https://docs.github.com/en/enterprise-cloud@latest/admin/policies/enforcing-policies-for-your-enterprise/enforcing-policies-for-github-actions-in-your-enterprise#allowing-select-actions-and-reusable-workflows-to-run) to learn more.
+
+You will need to allow the following GitHub Actions for Gruntwork Pipelines to run successfully:
+- [gruntwork-io/pipelines-dispatch](https://github.com/gruntwork-io/pipelines-dispatch)
+- [gruntwork-io/pipelines-orchestrate](https://github.com/gruntwork-io/pipelines-orchestrate)
+- [gruntwork-io/pipelines-execute](https://github.com/gruntwork-io/pipelines-execute)
+- [gruntwork-io/terragrunt-action](https://github.com/gruntwork-io/terragrunt-action)
+- [aws-actions/configure-aws-credentials](https://github.com/aws-actions/configure-aws-credentials)
+- [Codex-/return-dispatch](https://github.com/Codex-/return-dispatch)
+- [Codex-/await-remote-run](https://github.com/Codex-/await-remote-run)
+- [dsaltares/fetch-gh-release-asset](https://github.com/dsaltares/fetch-gh-release-asset)
+- [peter-evans/create-pull-request](https://github.com/peter-evans/create-pull-request)
+
+Gruntwork recommends using a tagged version of the `gruntwork-io` actions and a wildcard for the non-Gruntwork actions, as the non-Gruntwork actions are wrapped in the Gruntwork actions. This will require less administrative work to keep policies up to date with versions being used by Gruntwork actions.
```

Comment:
Worth including a copy-pastable example here?

Reply:
I considered it, but since the versions we are recommending will semi-regularly change, i ultimately opted against it so we wouldn't need to keep it up to date in the docs. If you think it would be helpful I'm happy to add it!

Reply:
I think it could be helpful, we can use a placeholder for the version number so its generic.

Reply:
+1 on the copy-pastable example. If it's cumbersome for us to maintain and keep this doc up to date, it would also be cumbersome for customers to keep up to date, so that's an issue we should directly address.

> Gruntwork recommends using a tagged version of the `gruntwork-io` actions and a wildcard

Can we explicitly recommend a specific tag they should use? As a side note, I've noticed our configs reference `@v1.X.Y` of an action, but should we just set them to `@v1` so that we can keep publishing updates and customers automatically get the latest?

Reply:
Added in [b3d5092](https://github.com/gruntwork-io/docs/pull/1148/commits/b3d50920f24c1da69e2e27df60fbb3bc44b75f09)

Reply:
Thanks! Approved.

Reply:
>  should we just set them to @v1 so that we can keep publishing updates and customers automatically get the latest

The `@v1.x.x` or `@v1` correspond to tags/releases in the repository. As far as I am aware you have to be explicit about the tag. You can't just say `anything under v1`.

Reply:
Oh got it. If they require @v1.x.x tagging then so be it!